### ISSUE TITLE
WebDriverException: Message: Can not connect to the Service geckodriver

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#102]
Zeus version:
`1.0.48`

Firefox version:
`Version: (52.3)`

Geckodriver version:
`geckodriver-v0.19.0-linux64.tar.gz`

Error info:
```Traceback (most recent call last):
  File "/root/Tools/Zeus-Scanner-master/var/google_search/search.py", line 242, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/root/Tools/Zeus-Scanner-master/var/google_search/search.py", line 129, in get_urls
    browser = webdriver.Firefox(profile, proxy=proxy_to_use)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/webdriver.py", line 144, in __init__
    self.service.start()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/common/service.py", line 102, in start
    raise WebDriverException("Can not connect to the Service %s" % self.path)
WebDriverException: Message: Can not connect to the Service geckodriver
````

Running details:
`Linux-4.12.0-kali2-amd64-x86_64-with-Kali-kali-rolling-kali-rolling`

Commands used:
`zeus.py -l dorks.txt xcarmultimarcas.com.br`

Log file info:
```2017-10-15 11:49:01,909;zeus-log;INFO;log file being saved to '/root/Tools/Zeus-Scanner-master/log/zeus-log-12.log'...
2017-10-15 11:49:01,909;zeus-log;INFO;using default search engine...
2017-10-15 11:49:01,910;zeus-log;INFO;starting dork scan with query 'admin account info" filetype:log'...
2017-10-15 11:49:01,910;zeus-log;INFO;attempting to gather query URL...
2017-10-15 11:49:02,028;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-15 11:49:02,028;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-15 11:49:51,551;zeus-log;ERROR;[7;31;31msearch.pyc failed to gather the URL from search engine, caught exception 'Message: Can not connect to the Service geckodriver' exception has been logged to current log file...
Traceback (most recent call last):
  File "/root/Tools/Zeus-Scanner-master/var/google_search/search.py", line 242, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/root/Tools/Zeus-Scanner-master/var/google_search/search.py", line 129, in get_urls
    browser = webdriver.Firefox(profile, proxy=proxy_to_use)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/webdriver.py", line 144, in __init__
    self.service.start()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/common/service.py", line 102, in start
    raise WebDriverException("Can not connect to the Service %s" % self.path)
WebDriverException: Message: Can not connect to the Service geckodriver


2017-10-15 11:51:21,642;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-15 11:51:21,643;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-15 11:51:21,643;zeus-log;INFO;[32mextracting traceback from log file...[0m
2017-10-15 11:51:21,644;zeus-log;INFO;[32mattempting to get firefox browser version...[0m
```

**Comment by Ekultek** [comment]
Wrong geckodriver, hang on

**Comment by Ekultek** [comment]
This is fixed via 5375c15. To be extra sure you should update firefox, reinstall Zeus and run `chmod a+x /usr/bin/geckodriver`. That was it's for sure the correct version, and for sure executable.
